### DISCUSSION TITLE
fine tuning a model?

### DISCUSSION THREAD

**Discussion by funkenpedro** [deepgram/community#480] [General help] [answered]
I've been testing Deepgram transcription, it misses place names and some other words that are common in the dialog im trying to capture but not so much in everyday use. How does one fine tune a model to recognize the place names and specific words?

**Comment by team-deepgram** [comment]
Thanks for asking your question about Deepgram! If you didn't already include it in your post, please be sure to add as much detail as possible so we can assist you efficiently, such as:
- The `request_id` if you have a question about your requests or transcription responses.
- The features you used or the full `api.deepgram.com` URL you sent your request to, including parameters.
- Any code snippets you can share.

**Comment by jkroll-deepgram** [comment, marked as answer]
Hi @funkenpedro, I recommend our [Keywords](https://developers.deepgram.com/docs/keywords) feature to help improve recognition of specific commonly-used names or entities that you're noticing being transcribed.

If you're seeing generally not as much accuracy as you'd like, you can also see which [models are available](https://developers.deepgram.com/docs/models-languages-overview) for the language(s) you're using. We recommend `nova-2-general` in most cases, and `nova-2-phonecall` for lower-quality/phonecall audio, and we have other domain-specific models for medical, finance, etc. Some customers like our `enhanced` models for entity accuracy as well.
